src/change/sdp.rs

```diff
@@ -1002,25 +1002,58 @@ fn update_media(
     // SSRC changes
     // This will always be for ReceiverSource since any incoming a=ssrc line will be
     // about the remote side's SSRC.
-    let infos = m.ssrc_info();
+    let infos = m.ssrc_info(false);
 
     let main = infos.iter().filter(|i| i.repairs.is_none());
 
-    if m.simulcast().is_none() {
-        // Only use pre-communicated SSRC if we are running without simulcast.
-        // We found a bug in FF where the order of the simulcast lines does not
-        // correspond to the order of the simulcast declarations. In this case
-        // it's better to fall back on mid/rid dynamic mapping.
+    // Use pre-communicated SSRCs if available and prefer them over dynamically signalled SSRCs via
+    // rid and repaired-rid. There's a bug in Firefox that causes it to emit incorrect mappings via
+    // rid/repaired-rid while the values signalled in SDP are correct.
 
-        for i in main {
-            // TODO: If the remote is communicating _BOTH_ rid and a=ssrc this will fail.
-            info!("Adding pre-communicated SSRC: {:?}", i);
-            let repair_ssrc = infos
-                .iter()
-                .find(|r| r.repairs == Some(i.ssrc))
-                .map(|r| r.ssrc);
-            streams.expect_stream_rx(i.ssrc, repair_ssrc, media.mid(), None);
+    for i in main {
+        // TODO: If the remote is communicating _BOTH_ rid and a=ssrc this will fail.
```

Comment:
I think we need to forbid the rid and repaired-rid in question from causing a change in SSRC mappings. I started on this below but wanted to run it past you first @algesten